content/Software Support/IDE Settings/Change-the-language-in-Arduino-IDE.md

```diff
@@ -3,8 +3,6 @@ title: "Change the language in Arduino IDE"
 id: 4403365287826
 ---
 
-> **Note:** Community translations were added to Arduino IDE 2 in version **2.0.0-rc6**. You can find the latest version on the [Software page](https://www.arduino.cc/en/software). <!-- TEMP_NOTICE: 22-04-27 -->
```

Comment:
I suggest also removing the other section about this problem from the article:

https://github.com/arduino/help-center-content/blob/155f4b43dfc7d5a9d7f5230fc65885c355566244/content/Software%20Support/IDE%20Settings/Change-the-language-in-Arduino-IDE.md?plain=1#L52-L57

Enough time has passed since the 2.0.0-rc6 release that it is no longer necessary.